.travis.yml

```diff
@@ -8,7 +8,8 @@ sudo: false
 
 # use linux as operating system
 os:
-  - linux
+  - osx
+    osx_image: xcode10.1
```

Comment:
I am nitpicking but these can be simplified using:

```
os: osx
osx_image: xcode10.1
```

Same goes for `jdk: openjdk11`